cmd/helm-sync/main.go

```diff
@@ -36,6 +36,14 @@ var (
 		"the name of the helm chart being synced")
 	flVersion = flag.String("version", os.Getenv(reconcilermanager.HelmChartVersion),
 		"the version of the helm chart being synced")
+	flValuesFiles = flag.String("values-files", os.Getenv(reconcilermanager.HelmValuesFiles),
+		"helm values files url")
```

Comment:
Is this local path or remote url? This isn't consistent with the R*Sync API field description.

Reply:
Now I think it should be better to be a url, this way user will have more flexibility and they can also use public values files, instead of must use a values file in the same repo. The downside is that, if user use the values file in the same repo, they need to write the full url again. If you agree, I can update the API description to be consistent.

Reply:
Users should be able to specify either local file path or remote url, and the tool should be able to detect it, instead of requiring local file path to be written in the url format. For example, Kustomize Helm plugin takes either a local file path or a remote URL to a values file.

If the usage of a field is changed here, the API doc needs to be updated in the same commit.

Reply:
https://docs.google.com/document/d/1lDMU0lulNYhIQ_8nk330r6eI72jp6_95qufTXy-LwR4/edit?resourcekey=0-YP0D5bF2S0bCJhnXaBeV-A&disco=AAAAdL1P-gY. Had a discussion with Morten before, since user cannot put the values file into our container directly, we will only support remote URL. I will update the API description to make it more clear.

Reply:
No, we shouldn't ask users to put values file in the container. For local file path - can users put multiple values files in the chart, and then reference different values files (as local file path in the remote chart) at runtime?

Reply:
So users don't really have a way to provide values files to the container without including them in the Helm charts. But I'm not sure if we have good use-cases why users would want to do that.
Remote files are possible, but I'm a bit skeptical about that one too since that in a lot of cases will require auth, which is likely to need different credentials from fetching the chart itself. Asking users to provide multiple sets of credentials gets kinda complicated.

My take on this is that we should not support values files in this way for now. We can let users provide the values through the API directly and we can revisit this if we have a good use-case. It is easier to leave it out than trying something and getting it wrong.

Reply:
My thought of the local values file usage is that users would include multiple values file in a chart, and specify different files in different clusters/environment at runtime. I'm not 100% sure if that's how users would use it, though. I think it's fine if we don't support it in the first version.

Reply:
My idea is that we can support the valuesfiles under the same repo which need Auth, or under different public repo. I changed the API field description to: 
// valuesFiles is a list of URL to Helm value files.
// Values files must be in the same repository with the Helm chart or in a public repository.

Reply:
And in the future we can decide if we want to support local file path.

Reply:
Will remove it in separate PR.